src/libstd/keyword_docs.rs

```diff
@@ -1100,10 +1100,25 @@ mod trait_keyword {}
 //
 /// A value of type [`bool`] representing logical **true**.
 ///
-/// The documentation for this keyword is [not yet complete]. Pull requests welcome!
+/// Logically `true` is not equal to [`false`].
+///
+/// ## Control structures that check for **true**
+///
+/// Several of rust's control structures will check for a `bool` argument evaluating to **true**.
```

Comment:
```suggestion
/// Several of Rust's control structures will check for a `bool` argument evaluating to **true**.
```

Matches other capitalization.